sections/semantics-embedded-content.include

```diff
@@ -5676,6 +5676,10 @@ attribute's value is a type that a <a>plugin</a> supports, then the value of the
   The <{video}> element is a <a>media element</a> whose <a>media data</a> is ostensibly video
   data, possibly with associated audio data.
 
+  Because the <{video}> element is necessary to support accessibility for audio content,
+  user agents should support playback of the same set of audio formats in the <{video}> element
```

Comment:
Should this reference "audio formats **and container formats** in the <{video}> element"?

Reply:
Yes, that seems reasonable.